kmip/services/server/kmip_server.py

```diff
@@ -53,6 +53,7 @@ def serve(self):
         self.socket.listen(0)
         while True:
             connection, address = self.socket.accept()
+            self.logger.info('New connection')
```

Comment:
If adding a logging statement, is it possible to include more detailed information about the connection that would be useful when debugging?